gb-ppu/src/drawing/state.rs

```diff
@@ -82,7 +84,7 @@ impl State {
         };
         self.step = (self.step + 1) % Self::STEP_COUNT;
         if self.step == 0 {
-            self.line = (self.line + 1) % Self::LINE_COUNT;
+            self.line = (self.line + 1) % (Self::LINE_COUNT);
```

Comment:
```suggestion
            self.line = (self.line + 1) % Self::LINE_COUNT;
```
parentheses not required here